modules/Hoot/ui/navbar.js

```diff
@@ -44,49 +46,83 @@ export default class Navbar {
             .classed( 'pointer hoot-logo', true );
 
-        this.menuButton = leftContainer
-            .insert( 'div', '.logo-container' )
-            .classed( 'menu-button button text-light pointer icon-container', true )
-            .on( 'click', () => this.toggleManagePanel() );
-
-        this.menuButton
-            .append( 'i' )
-            .classed( 'medium material-icons', true )
-            .text( 'menu' );
+        if ( this.isLoggedIn ) {
+            this.menuButton = leftContainer
+                .insert( 'div', '.logo-container' )
+                .classed( 'menu-button button text-light pointer icon-container', true )
+                .on( 'click', () => this.toggleManagePanel() );
+
+            this.menuButton
+                .append( 'i' )
+                .classed( 'medium material-icons', true )
+                .text( 'menu' );
+        }
     }
 
     createRightContainer() {
         let rightContainer = this.navbar
             .append( 'div' )
             .classed( 'nav-item', true );
 
-        let dropdownContent = rightContainer
-            .append( 'ul' )
-            .classed( 'dropdown-content fill-white', true );
-
-        dropdownContent
-            .append( 'li' )
-            .classed( 'dropdown-item pad2x strong pointer', true )
-            .append( 'a' )
-            .attr( 'href', '#!' )
-            .text( 'About' );
-
-        let dropdownToggle = rightContainer
-            .append( 'div' )
-            .classed( 'about-toggle icon-container button flex align-center text-light pointer', true )
-            .on( 'click', () => this.openAboutModal() );
-
-        dropdownToggle
-            .append( 'i' )
-            .classed( 'medium material-icons', true )
-            .text( 'info_outline' );
-
-        //dropdownToggle
-        //    .append( 'i' )
-        //    .classed( 'medium material-icons', true )
-        //    .text( 'arrow_drop_down' );
-
-        //this.initDropdown();
+        if ( this.isLoggedIn ) {
+            let user = JSON.parse( localStorage.getItem( 'user' ) );
+
+            let dropdownToggle = rightContainer
+                .append( 'div' )
+                .classed( 'dropdown-toggle icon-container button flex align-center text-light pointer', true );
+
+            dropdownToggle
+                .append( 'i' )
+                .classed( 'medium material-icons', true )
+                .text( 'info_outline' );
+
+            dropdownToggle
+                .append( 'i' )
+                .classed( 'medium material-icons', true )
+                .text( 'arrow_drop_down' );
+
+            let dropdownContent = rightContainer
+                .append( 'ul' )
+                .classed( 'dropdown-content fill-white', true );
+
+            dropdownContent
+                .append( 'li' )
+                .classed( 'dropdown-item pad2x strong', true )
+                .append( 'span' )
+                .text( 'Logged in as ' + user.display_name );
```

Comment:
I personally like the display name shown in the nav bar like Hoot v1.  Not sure if that opinion is shared by anyone else.

Reply:
subject to change. it wouldn't take long to move back out to the navbar, but i think it took up a lot of unnecessary space. maybe it can display in the navbar briefly when the user logs in or opens the page, and then fades out?

Reply:
I'm thinking of testing with users.  I now have two users and have a habit of glancing up at who I'm logged in as to make sure I'm seeing the correct folders/datasets.